### ISSUE TITLE
bndrun window in maven-layout does not show bundles (e 2022-9)

### ISSUE THREAD

**Issue by stbischof** [bndtools/bnd#5409]
When opening the Resolve/Run (bndrun) window in eclipse 2022-9 in a project that uses mvn layout you could not longer see the MavenDependencies in the left site of the window.

Left: Eclipse 2022-9        Right: Eclipse 2022-3
![Screenshot from 2022-10-25 14-33-43](https://user-images.githubusercontent.com/33224746/197774097-2e438532-0a9d-4e56-9b80-bf0115362b3d.png)

**Comment by bjhargrave** [comment]
Isn't this addressed by the recent PRs from @timothyjward?

**Comment by timothyjward** [comment]
> Isn't this addressed by the recent PRs from @timothyjward?

Yes and no. They appear, but then disappear again after < 1 second. The recent PRs make the resolve work though, which was completely broken before.

**Comment by stbischof** [comment]
on newest snapshot this came up;
```
An internal error occurred during: "Updating Maven Project".
Receiver class bndtools.m2e.MavenDependenciesRepository does not define or inherit an implementation of the resolved method 'abstract void mavenProjectChanged(java.util.List, org.eclipse.core.runtime.IProgressMonitor)' of interface org.eclipse.m2e.core.project.IMavenProjectChangedListener.
```

**Comment by timothyjward** [comment]
This is probably why the panel paints then disappears on Eclipse 22-06+

**Comment by timothyjward** [comment]
> fixed: mavenProjectChanged method by #5429
> 
> now i found this in error log
> 
> ```
> org.osgi.framework.BundleException: Could not resolve module: bndtools.m2e [439]
>   Unresolved requirement: Import-Package: org.apache.maven.artifact; bundle-symbolic-name="org.eclipse.m2e.maven.runtime"; bundle-version="[1.16.0,2.0.0)"; provider="m2e"
> 
> 	at org.eclipse.osgi.container.Module.start(Module.java:463)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1852)
> 	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1845)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1786)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1750)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1672)
> 	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
> 	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
> 	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
> ```

This was fixed in https://github.com/bndtools/bnd/commit/0cc2c278818cc6fc28ed3d17bb2c22e4a77f2b5c

I suggest trying to install the latest snapshot, which should be working now

**Comment by stbischof** [comment]
sorry did not switch branche. works fine. thanks
